test/e2e/tests/errors.spec.js

```diff
@@ -23,6 +23,36 @@ describe('Sentry errors', function () {
       },
     ],
   };
+  it('should NOT send error events when participateInMetaMetrics is false', async function () {
+    await withFixtures(
+      {
+        fixtures: new FixtureBuilder()
+          .withMetaMetricsController({
+            metaMetricsId: null,
+            participateInMetaMetrics: false,
+          })
+          .build(),
+        ganacheOptions,
+        title: this.test.title,
+        failOnConsoleError: false,
+        testSpecificMock: mockSentry,
+      },
+      async ({ driver, mockedEndpoint }) => {
+        await driver.navigate();
+        await driver.fill('#password', 'correct horse battery staple');
+        await driver.press('#password', driver.Key.ENTER);
+        // Trigger error
+        driver.executeScript('window.stateHooks.throwTestError()');
+        driver.delay(3000);
```

Comment:
Question: is there a way to bypass the delay here?

Reply:
No we need to wait sometime to give time for a network request to occur

Reply:
In the inverse test, we wait for a specific network request. I want to give a similar pause but we can't wait for anything in particular.